doc/api/util.md

```diff
@@ -207,6 +207,23 @@ Values may supply their own custom `inspect(depth, opts)` functions, when
 called these receive the current `depth` in the recursive inspection, as well as
 the options object passed to `util.inspect()`.
 
+### util.inspect.config([options])
```

Comment:
Perhaps a `defaultOptions` setter/getter would be better here? Something like...

```js
util.inspect.defaultOptions = options;

// or even...

util.inspect.defaultOptions.maxArrayLength = 1000;
```